internal/livetiming/session.go

```diff
@@ -0,0 +1,67 @@
+package livetiming
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+	"time"
+)
+
+// Sessions contains all possible sessions in a two-depth map. The map is map[year]map[round]Session
+type Sessions map[int64]map[int64]Session
+
+type Session struct {
+	Season int64
+	Round  int64
+	Name   string
+	Date   time.Time
+	Path   string
+}
+
+func (s *Session) UnmarshalJSON(b []byte) error {
+	var (
+		rawStr string
+		ok     bool
+		err    error
+	)
+
+	raw := make(map[string]interface{})
+
+	if err = json.Unmarshal(b, &raw); err != nil {
+		return err
+	}
+
+	if rawStr, ok = raw["season"].(string); !ok {
+		return errors.New("could not unmarshal season field of session")
+	}
+	s.Season, err = strconv.ParseInt(rawStr, 10, 64)
```

Comment:
similarly you'll mostly see `strconv.Atoi` in our codebase except in very specific circumstances